src/Compilers/Helpers/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Loads analyzer assemblies from their original locations in the file system.
+    /// Assemblies will only be loaded from the locations specified when the loader
```

Comment:
I suspect `Assembly.LoadFrom` in that particular case would also load the NGEN images for the IDE analyzer assemblies as those assemblies must be in the binding path for `Assembly.Load` to work, and as such the LoadFrom would be promoted to a Load.

However, since it is working as-is, I don't intend to change anything. :-)